help/en/releasenotes/current-draft-note.shtml

```diff
@@ -411,6 +411,10 @@
           <li>The ConditionalNG editor and the Module editor now have the option
           <i>Paste copy</i>. With it, you can paste from the clipboard as many times
           as needed.</li>
+          <li>Adds support of multiple types of CSV files to LogixNG Tables
```

Comment:
A dot is missed at the end of this sentence.

Reply:
Done